benchmark/nixlbench/contrib/Dockerfile.aws-amzn2023

```
@@ -0,0 +1,249 @@
+    gcc-c++ \
+    git \
+    numactl \
+    numactl-devel \
+    hwloc \
+    hwloc-libs \
+    hwloc-devel \
+    openssl-devel
+
+WORKDIR /workspace
+
+RUN curl --retry 3 --retry-delay 2 -fsSL -o aws-efa-installer-${EFA_VERSION}.tar.gz \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)